draft-ietf-quic-transport.md

```diff
@@ -5374,6 +5379,14 @@ CRYPTO_ERROR (0x1XX):
 
 See {{iana-error-codes}} for details of registering new error codes.
 
+In defining these error codes, several principles are applied.  Error conditions
+that might require specific action on the part of a recipient are giving unique
```

Comment:
s/giving/given

Reply:
I'm almost surprised that I can update your comment, turn it into a suggestion, then cause a commit to be created under your name.  The power is going to my head...

Reply:
I'd have made the suggestion myself but I'm on a mobile device that hides that widget. Good to know we have a workflow that makes my life easier ;)